canarytests/agent/src/main/java/emf/canary/ECSRunnable.java

```diff
@@ -21,23 +24,32 @@ public void run() {
         }
         Configuration config = EnvironmentConfigurationProvider.getConfig();
         config.setLogGroupName("/Canary/Java/CloudWatchAgent/Metrics");
-        logger.setNamespace("Canary");
-        logger.setDimensions(
-                DimensionSet.of(
-                        "Runtime", "Java8",
-                        "Platform", "ECS",
-                        "Agent", "CloudWatchAgent",
-                        "Version", version));
+
+        try {
+            logger.setNamespace("Canary");
+            logger.setDimensions(
+                    DimensionSet.of(
+                            "Runtime", "Java8",
+                            "Platform", "ECS",
+                            "Agent", "CloudWatchAgent",
+                            "Version", version));
+        } catch (InvalidNamespaceException | InvalidDimensionException e) {
+            System.out.println(e);
```

Comment:
very nit: Just noticed we import `slf4j` in the project, what do you think of using `log.error()` instead of `system`?

Reply:
I could yes. I'll skip adding the import statement and initializing the logger since these code blocks are examples and don't need to run.

Reply:
This is just for the canary, so I don't think it is necessary.